faunadb-java-dsl/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -995,10 +1086,20 @@ public static Expr Date(Expr str) {
    *
    * @see <a href="https://fauna.com/documentation/queries#misc_functions">FaunaDB Miscellaneous Functions</a>
    */
+  @Deprecated
```

Comment:
Isn't it missing the deprecation message telling to use `NewId` instead?

Reply:
There's no way to specify a message on this annotation, I can do this in javadoc if needed

Reply:
Yeah. A javadoc would be nice. I _think_ it has a tag for it. I'm not sure.